log4j-kafka/src/main/java/org/apache/logging/log4j/kafka/appender/KafkaAppender.java

```diff
@@ -83,7 +83,7 @@ public boolean isSyncSend() {
             return syncSend;
         }
 
-        public boolean isSendTimestamp() { return sendTimestamp; }
+        public boolean isSendEventTimestamp() { return sendEventTimestamp; }
```

Comment:
This needs to be split into >1 line to match the coding style.

Reply:
Fixed.